draft-ietf-httpbis-incremental.md

```diff
@@ -100,13 +100,20 @@ intermediaries to start forwarding the message downstream before the entire
 message is received.
 
 The Incremental header field is defined as a structured field
-{{STRUCTURED-FIELDS}} of type Item.  There is just one valid value, which is of
-type Boolean: "?1".
+{{STRUCTURED-FIELDS}} of type Item.
+Only Boolean values ({{Section 3.3.6 of STRUCTURED-FIELDS}}) are valid;
+a recipient ignores the field if it contains any other type.
 
 ~~~
-Incremental = ?1
+Incremental = sf-boolean
```

Comment:
nit: you don't really need the ABNF here and its against the style guide but I suspect someone will catch that later anyway; see https://httpwg.org/admin/editors/style-guide#structured-fields